### PR TITLE
Implement new zombie: Zapper zombie

### LINKED ISSUE #35933: Replace early spawning shockers with new electric zombie
**Is your feature request related to a problem? Please describe.**  
Shockers are disproportionately dangerous compared to other monsters that can spawn at the start of the game now.

**Describe the solution you'd like**  
Add a new electric zombie that's essentially a shocker without the lightning attack. Make this spawn at the start of the game then evolve into the current shockers.

**Additional context**  
#32131 is related.

### PR DESCRIPTION
**Pull request by arijust**
#### Summary

```SUMMARY: Content "Implement new zombie: Zapper zombie"```
#### Purpose of change

Close #35933
#### Describe the solution

Zapper zombie is my suggestion for early game electric zombie.
It doesn't have ranged electric attack, but still can use zapback on melee hit.
Overall, it's slightly more durable than normal zombie.
#### Describe alternatives you've considered

Lower spawn rates for shocker_zombie.

#### Additional context

While not directly related to PR, I wanted to mention two things.
1. monsters.json seems to be bloated and some monsters can be easily moved into other files.
2. What about Ice/Cold-themed zombies? I am aware it would certain amount of work on C++ part.

### CODE REVIEW

**Comment by thethunderhawk** [comment]
If this thing is supposed to evolve into a Shocker, it ought to have harvestable CBMs as well yeah?

**Comment by Amoebka** [comment]
Neither name nor description make it clear enough that "static" refers to electricity rather than lack of movement. For a zombie that's intended to be an ealy game player killer that's very dickish.

**Comment by FuelType-Memes** [comment]
> Neither name nor description make it clear enough that "static" refers to electricity rather than lack of movement. For a zombie that's intended to be an ealy game player killer that's very dickish.

Maybe "Zapper zombie" for them additional alliteration points?

**Comment by Fris0uman** [comment]
I'd say yes to zapper zombie, also shouldn't it be `the worst case of static hair [...]`

**Comment by Zireael07** [comment]
+1 to name change, I was very confused when scrolling through the PR list and wondering why in the hell someone would go to the trouble of adding an umoving zombie type xDDD

File: data/json/monsters/monsters.json [hunk 1]
```diff
@@ -3846,6 +3846,51 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_zombie_static",
+    "type": "MONSTER",
+    "name": "zapper zombie",
+    "description": "Very pale dead body, it has the worst case of static hair you have ever seen.  Sometimes, you can see jolts of electricity around it.",
```
Comment (matskuman5) [inline data/json/monsters/monsters.json]:
```suggestion
    "description": "A very pale dead body with the worst case of static hair you have ever seen.  Sometimes, you can see jolts of electricity around it.",
```

**Comment by kevingranade** [comment]
Oh we will be adding unmoving zombies later, once they can lie down in underbrush and remain mostly unseen.

Definitely no need to add CBM drops to these, we are planning on reversing that bit of life with shocker zombies anyway.

**Comment by Night-Pryanik** [comment]
If there are no CBMs in this new monster, then we need to justify its ability to generate electricity.

**Comment by kevingranade** [comment]
It's a magic space zombie.

**Comment by arijust** [comment]
I would rather say its ability is similar to electric eel.

**Comment by SirPendrak** [comment]
I would say its better to either all zombies from shocker line have CBMs, or none of them having bionics.  
Situation where most of them have CBMs but the most basic one doesnt is strange at least and can lead to strange situations like players intentionally leaving zappers allive so they can evolve and drop that sweet loot. Also for fluff consistency, eihter all of them have their powers from CBMs, or all of them use some kind of bio-chem-magic.

**Comment by arijust** [comment]
Ok.
I will get it done with all zombies on Zapper evo line, then.

**Comment by anothersimulacrum** [comment]
Might be better to leave yours with no CBM harvest and the rest unchanged, then address that in a separate PR.
Because that change will require some discussion also probably adding CBMS to spawn in normal zombies with a low chance.

**Comment by SirPendrak** [comment]
For now IMHO best option would be to give all of them the same bionics. I am not trying to oppose Kevin and his words about removing bionics from shocker line, but that should be a separate PR with a wider rebalance/rework. For now removing CBMs from all of them or leaving most basic one without CBM would be half-assed state. 
That said, its strange that currently shocker brute and incadescent husk have different CBM drops than normal shocker, should be the same group for now.

**Comment by kevingranade** [comment]
Removing the CBMs is fine, adding CBM drops to this new monster is not, leaving the overall CBM situation unchanged is also fine.

**Comment by FuelType-Memes** [comment]
> 
> It's a magic space zombie.

Damn! I always thought shocker's powers came from faulty bionics, boomer's from too many beans, spitter's from too much hot ramen and scientist's from the MIT degree

**Comment by arijust** [comment]
>leaving the overall CBM situation unchanged is also fine.

It was left unchanged. Zapper zombie has standard zombie harvest.

File: data/json/monsters/monsters.json [hunk 2]
```diff
@@ -3846,6 +3846,51 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_zombie_static",
+    "type": "MONSTER",
+    "name": "zapper zombie",
+    "description": "A very pale dead body with the worst case of static hair you have ever seen.  Sometimes, you can see jolts of electricity around it.",
```
Comment (kevingranade) [inline data/json/monsters/monsters.json]:
```suggestion
    "description": "A very pale dead body with the worst case of static hair you have ever seen.  Sometimes, you can see sparks of electricity around it.",
```

**Comment by SirPendrak** [comment]
If all electric zombies are space zombies, then following that logic, shouldn't they be disallowed to spawn at day one at all? Because then even theese weaker Zappers are already "unnatural", evolved zombie.

**Pull request merged**